### PR TITLE
Console docker integration tests

### PR DESCRIPTION
**Pull request by groldan**
Using `fabric8io/docker-maven-plugin` to set up test environment.

See README.md for more information.

### CODE REVIEW

**Comment by groldan** [comment]
Note this PR is not ready to be merged yet. Issuing it for review and discussion.

**Comment by groldan** [comment]
@pmauduit @fvanderbiest yay! it worked on travis too! check this out: https://travis-ci.org/georchestra/georchestra/jobs/483690940#L2468

**Comment by groldan** [comment]
Integration tests have been moved to the `it` maven profile and enabled on travis. Run `mvn -Pit verify` locally to have them run.

**Comment by groldan** [comment]
Fixes #2305 on master

File: console/src/it/java/org/georchestra/console/integration/RolesIT.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+	private @Autowired LdapTemplate ldapTemplateSanityCheck;
+
+	private @Value("${ldap_port}") int ldapPort;
+
+	private @Value("${psql_port}") int psqlPort;
+
+	public @Rule TestName testName = new TestName();
+
+	RestTemplate restTemplate = new RestTemplate();
```
Comment (cmangeat) [inline console/src/it/java/org/georchestra/console/integration/RolesIT.java]: What is the purpose of restTemplate ?

Comment (groldan) [reply console/src/it/java/org/georchestra/console/integration/RolesIT.java]: None, just an oversight from a previous test. Removed it, good catch.

**Pull request merged**